body/src/main/java/com/networknt/body/BodyHandler.java

```diff
@@ -69,7 +69,7 @@
     private volatile HttpHandler next;
 
     public BodyHandler() {
-
+        if(logger.isInfoEnabled()) logger.info("BodyHandler is loaded.");
```

Comment:
any reason why this should be `INFO` as opposed to `DEBUG` ?

Reply:
A handler is a singleton and only loaded once, that is why we make it as info in case project team want to double check what kind of handlers are loaded as they are all plugins. If you see there is no issue, please approve it so that we can merge it. We need two reviewers to ensure the quality of the code. Thanks.